src/components/pages/__tests__/AlderStep.test.tsx

```diff
@@ -42,47 +44,123 @@ beforeEach(() => {
     {},
     { validateFields: mockValidateFields, clearError: mockClearError },
   ])
+  ;(useFieldChange as jest.Mock).mockReturnValue({
+    handleFieldChange: mockHandleFieldChange,
+  })
 })
+
```

Comment:
Det kan se ut som at det mangler test for når Textfield for foedselAar er tomt. Jeg ser at det er kodet en default verdi på 0:
`it.target.value === '' ? 0 : parseInt(it.target.value, 10)`

Det samme ser ut til å gjelder for Textfield for inntektOver1GAntallAar lengre ned.